packages/searchbase/src/schema.js

```diff
@@ -0,0 +1,307 @@
+import { componentsAlias, backendAlias, dataTypes } from './utils';
+
+const { SEARCHBASE, SEARCHBOX, SEARCHCOMPONENT } = componentsAlias;
+const { MONGODB, ELASTICSEARCH } = backendAlias;
+
+export default {
+  url: {
+    components: [SEARCHBASE],
+    backend: [ELASTICSEARCH, MONGODB],
+    type: dataTypes.STRING,
+    required: true
+  },
+  index: {
+    components: [SEARCHBASE, SEARCHBOX, SEARCHCOMPONENT],
+    backend: [ELASTICSEARCH, MONGODB],
+    type: dataTypes.STRING,
```

Comment:
@mohdashraf010897 
1. As we discussed, schema has to be different for each BE. How would you handle the following error?
- `index` is required for mongodb but not for ES
2. Why do you have to maintain `components` info in schema? I think it is redundant.